opm/parser/share/keywords/000_Eclipse100/R/ROCKNUM

```diff
@@ -0,0 +1 @@
+{"name" : "ROCKNUM" , "sections" : ["REGIONS"], "data" : {"value_type" : "INT"}}
```

Comment:
Would be very simple to add "full" support for `ROCKNUM` by adding an entry [here](https://github.com/OPM/opm-parser/blob/master/opm/parser/eclipse/EclipseState/Eclipse3DProperties.cpp#L188)